FrontEnd/src/colors.scss

```diff
@@ -4,4 +4,6 @@
 
     --white: #ffffff;
     --black: #333333;
+
+    --red: #db7093;
```

Comment:
this I guessed, since there's in specs for the component.
feel free to change it later on

Reply:
haha, it was a built-in color that VSCode gave when I've started to type "red".
it sortof looks maenta/pink actually, but again, feel free to change it